extension/app/src/components/conversation/ConversationContainer.tsx

```diff
@@ -207,20 +207,16 @@ export function ConversationContainer({
               <Page.SectionHeader title="Start a conversation" />
             </div>
             <div className="flex space-x-1">
-              <Tooltip
-                label={
+              <Button
```

Comment:
This was raising a js error even with a `tooltipTriggerAsChild` so replacing with a new tooltip props (does not work yet but WIP from Jules)

https://dust4ai.slack.com/archives/C050SM8NSPK/p1730882644879119